components/edit-collective/sections/CollectiveGoals.js

```diff
@@ -73,20 +77,16 @@ class CollectiveGoals extends React.Component {
       },
       {
         name: 'amount',
-        pre: getCurrencySymbol(props.currency),
+        pre: 'USD',
```

Comment:
If we want to display the currency as a three-letters code, we should use `props.currency` directly. `USD` should not be forced because some collectives operate with different currencies (ie. EUR, JPY...etc)

Reply:
Done. :+1: